spec/models/event_spec.rb

```diff
@@ -121,4 +121,23 @@
       end
     end
   end
+
+  describe '#upcoming?' do
+    subject(:event_upcoming?) { event.upcoming? }
+    context 'the event is in the future' do
+      let!(:event) do
+        FactoryGirl.create(:event, date: Time.zone.today + 1.hour)
```

Comment:
Can we get away with using `build` here? Might not get picked up by the scope, I guess.